thriftbp/errors.go

```diff
@@ -59,3 +61,25 @@ func BaseplateErrorFilter(codes ...int32) retrybp.Filter {
 		return next(err)
 	}
 }
+
+// NewBaseplateError is a helper function for creating baseplate.Error thrift
```

Comment:
What you wrote in your PR description I would expect here, something along the lines of:

```
NewBaseplateError is a helper function for creating baseplate.Error thrift objects to avoid manual type-conversion that the generated Thrift code requires.
```

Reply:
updated